custom/ewsghana/tasks.py

```diff
@@ -33,12 +37,18 @@
                   526, 4, 30, 1, 14, 23, 521, 532, 516, 461, 520, 525, 961, 641, 257, 348]
 
 
-# @periodic_task(run_every=timedelta(days=1), queue=getattr(settings, 'CELERY_PERIODIC_QUEUE', 'celery'))
+@periodic_task(run_every=crontab(hour="23", minute="55", day_of_week="*"),
+               queue=getattr(settings, 'CELERY_PERIODIC_QUEUE', 'celery'))
```

Comment:
@kkrampa how confident are you that this line is not going to cause memory or performance issues based on the current set of domains configured on prod?